File: src/main/java/com/bajdcc/LALR1/interpret/os/ui/UIMain.java
package com.bajdcc.LALR1.interpret.os.ui;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【界面】入口
 *
 * @author bajdcc
 */
public class UIMain implements IOSCodePage {
	@Override
	public String getName() {
		return "/ui/main";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/ui/UIMain.kt
package com.bajdcc.LALR1.interpret.os.ui

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【界面】入口
 *
 * @author bajdcc
 */
class UIMain : IOSCodePage {
    override val name: String
        get() = "/ui/main"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
